### PR TITLE
Add an accessible name for the package source lists for screen readers

### PR DESCRIPTION
**Pull request by jebriede**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/872

Regression? Last working version: N/A

## Description
Add an accessible name for both the package source and machine-wide package source lists in the Options -> Package Sources page for screen readers to read the focusable lists to the user.

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
Accessibility fix. Tested with Accessibility Insights. Tests passed on both list controls.
  - [x] Test exception
  - **OR**
  - [ ] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [x] N/A

### CODE REVIEW

File: src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx [hunk 1]
```diff
@@ -677,17 +680,20 @@
...
   <data name="MachineWideSourcesLabel.Location" type="System.Drawing.Point, System.Drawing">
-    <value>0, 157</value>
+    <value>0, 133</value>
   </data>
   <data name="MachineWideSourcesLabel.Margin" type="System.Windows.Forms.Padding, System.Windows.Forms">
     <value>0, 0, 0, 0</value>
   </data>
   <data name="MachineWideSourcesLabel.Size" type="System.Drawing.Size, System.Drawing">
-    <value>166, 23</value>
+    <value>321, 25</value>
```
Comment (zivkan) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: I'm very confused by this PR. Normally resx is for strings that need to be translated into different language. Having changes to size and location of controls seem out of scope of "add an accessible name for ...".

Comment (zivkan) [inline src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: Are the localization team able to see an example of the dialog/window for each language, and modify the value for each language?  If not, do you need to check every language to make sure the size is large enough to display the appropriate label?

Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: @zivkan: I noticed these changes as well and am confused by them as well. This is an auto-generated file that I did not change. I did not change the layout of any controls. I simply set an AccessibleName property on two lists and saved and this was the result. I reverted the changes and repeated the exact same process and it gave me the same changes. I don't want to manually modify this file as it's auto-generated and there are many parts of it I don't understand. This isn't the type of resx file I'm used to. Suggestions?

Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: I didn't actually change the size values. There's no reason this should have been updated by me simply setting an AccessibleName on two lists. This was automatically output when I clicked save. I repeated the process two times to ensure I didn't accidentally introduce a size change and it produced the same result.

Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: To add to this, it seems WinForms stores all UI element properties in the resx. It doesn't seem to be used exclusively for localization.

Comment (jebriede) [reply src/NuGet.Clients/NuGet.PackageManagement.UI/Options/PackageSourcesOptionsControl.resx:696]: @zivkan this is relevant: https://stackoverflow.com/questions/5543159/resource-files-get-modified-when-the-form-is-opened-in-visual-studio

It seems just opening the designer in Visual Studio on a machine with different hardware / OS will have the affect that some layout properties can change.

**Review by zivkan — APPROVED** [review]
Can you "just" rewrite the entire options page in WPF instead? 😛

**Comment by jebriede** [comment]
@zivkan I would sincerely love to.
